src/solver/src/simulation.C

```diff
@@ -309,6 +312,16 @@ namespace GRINS
     context.error_estimator = _error_estimator;
     context.print_qoi = _print_qoi;
 
+    if (_output_residual_sensitivities &&
+        !_forward_parameters.parameter_vector.size())
```

Comment:
Huh... yes, but not right away.  For some reason my code in libMesh is only creating permanent dR/dp vectors (well, retained until the user wants to get rid of them, anyway) in the forward case (and in some Hessian cases), but temporary vectors (deleted before the end of the calculation) in the adjoint case.  I'll need to refactor that before we can get at them.